pages/servers.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: page
+title:  "Servers"
+permalink: /servers/
+---
+
+List of servers (in alphabetical order) that provide a CCTray feed:
+
+### Buddybuild
+
+* Server home: <https://www.buddybuild.com/>
+* CCMenu setup instructions: <http://docs.buddybuild.com/docs/ccmenu>
+
+### CircleCI
+
+* Server home: <https://circleci.com/>
+* CCMenu setup instructions: <https://circleci.com/docs/polling-project-status>
```

Comment:
They've deprecated this link. We could link to their docs at: https://github.com/circleci/circleci-docs/blob/master/jekyll/_cci1/polling-project-status.md